src/main/java/org/cyclops/integratedrest/http/HttpServerHandler.java

```diff
@@ -100,7 +100,7 @@ private boolean writeResponse(HttpRequest request, ChannelHandlerContext context
                 HttpVersion.HTTP_1_1, responseStatus,
                 Unpooled.copiedBuffer(responseString, CharsetUtil.UTF_8));
 
-        response.headers().set(HttpHeaderNames.CONTENT_TYPE, "text/plain; charset=UTF-8");
+        response.headers().set(HttpHeaderNames.CONTENT_TYPE, "application/ld+json; charset=UTF-8");
```

Comment:
```suggestion
        response.headers().set(HttpHeaderNames.CONTENT_TYPE, "application/ld+json");
```

Reply:
Afaik the charset is actually redundant so it could definitely be removed